db/db_filesnapshot.cc

```diff
@@ -163,6 +163,10 @@ Status DBImpl::GetSortedWalFiles(VectorLogPtr& files) {
   return wal_manager_.GetSortedWalFiles(files);
 }
 
+Status DBImpl::GetCurrentWalFile(LogFile** current_log_file) {
+  return wal_manager_.GetWalFile(logfile_number_, current_log_file);
```

Comment:
Can you explain why it is safe to directly access `logfile_number_` without any synchronization mechanism?